docs.phtml

```diff
@@ -483,7 +483,7 @@ $older7		="3";
 <div class="featureborder">
 <h2 class="featuretitle"><?php echo T_("Older GnuCash Documentation");?></h2>
 </div>
-<p class="justify"><?php echo T_("This section contains all the older <span class=\"gnucash\">GnuCash</span> documentation. If you are using one of these old versions of <span class=\"gnucash\">GnuCash</span>, it is highly recommended that you upgrade to the latest stable version.");?></p>
+<p class="justify"><?php echo T_("This section contains all the older <span class=\"gnucash\">GnuCash</span> documentation. If you are using one of these old versions of <span class=\"gnucash\">GnuCash</span>, it is highly recommended you upgrade to the latest stable one.");?></p>
```

Comment:
Prefer to leave "that" in the sentence, although "one" is an improvement.

Reply:
In my reading into why that is, I found the whole thing to be a challenge.